util/imagetools/inspect.go

```diff
@@ -47,14 +51,19 @@ func (r *Resolver) resolver() remotes.Resolver {
 			}
 			for i := range res {
 				res[i].Authorizer = r.auth
+				res[i].Client = tracing.DefaultClient
 			}
 			return res, nil
 		},
-		Client: tracing.DefaultClient,
 	})
 }
 
 func (r *Resolver) Resolve(ctx context.Context, in string) (string, ocispec.Descriptor, error) {
+	// needs to discard containerd logger otherwise it will print
+	// unnecessary info during image reference resolution like
+	// https://github.com/containerd/containerd/blob/1a88cf5242445657258e0c744def5017d7cfb492/remotes/docker/resolver.go#L288
+	ctx = bxctx.WithLoggerDiscarded(ctx)
```

Comment:
needs to discard containerd logger otherwise it will print unnecessary info during image reference resolution:

```console
$ docker buildx imagetools inspect alpinefoo
INFO[0001] trying next host                              error="pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed" host=registry-1.docker.io
error: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

now:

```console
$ docker buildx imagetools inspect alpinefoo
error: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```